src/components/link/link.helper.js

```diff
@@ -68,10 +68,10 @@ function getRadiusStrategy(type) {
  * @returns {string} the path definition for the requested link
  * @memberof Link/helper
  */
-function buildLinkPathDefinition({ source = {}, target = {} }, type = LINE_TYPES.STRAIGHT) {
+function buildLinkPathDefinition({ source = {}, target = {}, linktype }, type = LINE_TYPES.STRAIGHT) {
```

Comment:
If you take my previous suggestion, you don't need to modify the function signature. The `type` is already passed in the second parameter, this function was already ready to receive a type from the exterior :)